src/main/java/com/vinylteam/vinyl/service/impl/DefaultPasswordRecoveryService.java

```diff
@@ -23,88 +23,106 @@
 @Slf4j
 @Service
 @RequiredArgsConstructor
-@PropertySource("classpath:application.properties")
 public class DefaultPasswordRecoveryService implements PasswordRecoveryService {
     private static final String RECOVERY_MESSAGE = "Hello, to change your password, follow this link: \n{applicationLink}/recoveryPassword/newPassword?token=";
-
```

Comment:
Could you please add line breaks here? So that blocks of fields are logically separated

`private static final String CONSTANT_FIELD = "value";

@Value("${some.value}" 
private String propertyValue;

private final Service service;`